__tests__/endpoints.test.js

```diff
@@ -0,0 +1,96 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const request = require("supertest");
+const app = require("../API/app");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("01 GET /api/topics", () => {
+  describe("Functionality", () => {
+    test("When correctly called, endpoint should return a 200 status code", () => {
```

Comment:
There is no need to split these tests. We're integration testing, not unit testing. I'd be expecting a 200 and data from request to return data to client.